### DISCUSSION TITLE
enhancement : adding webGL and GODOT-like webgame support, with multi-player capability, homebrew-repository

### DISCUSSION THREAD

**Discussion by freestateme** [stoatchat/discussions#321] [Feature Suggestions]
I think something that could turn Revolt into a killer app to slay discord is adding [webGL](https://get.webgl.org/) and [GODOT](https://godotengine.org/) - [like](https://pypi.org/project/WebPython/) webgame support, with multi-player capability, homebrew-repository with a server app integrator/manager, and maybe a proof of concept game like two+-player online multiplayer tetris-clone, battleship, or chess in 2D; the [GODOT](https://godotengine.org/) - [like](https://pypi.org/project/WebPython/) support would make a game maker/editor feature built in and could accelerate community improvements in small releases

I know that discord was born from gaming, and keeping some of that spirit alive with a web-game multiplayer session with friends and people in servers would be amazing.

I know that this, paired with the fact there is already video chat support, and text chat, means I could use it with my grandpa to enjoy a game together over the internet.

I guess file sharing @ email size would be nice, or even bigger; I guess there would be support for I2P web-torrent interface integration, but hey, I could go on about hypothetical features all day

if this is possible great, just sounds like a nice feature

I understand the focus is on a polished feeling, functional, interesting and enjoyable chat experience, and this takes low priority, but I hope it could make it into discussions about integrating some more hypothetical "nice to have" possible future features into the roadmap might make this project 11/10; hey, maybe nothing gets done about this request for #enhancement , but at least I have now put the idea out there.

I hope this inspires!

Best luck!

**Comment by ghost** [comment]
People can join games in any ROOM.
A game channel
This game channel is combined with all communities.

Chess
Poker
Go (game)
Mahjong
Shogi
Xiangqi
